spaceship/spec/portal/portal_stubbing.rb

```diff
@@ -1,7 +1,11 @@
 class PortalStubbing
   class << self
     def adp_read_fixture_file(filename)
-      File.read(File.join('spaceship', 'spec', 'portal', 'fixtures', filename))
+      File.binread(File.join('spaceship', 'spec', 'portal', 'fixtures', filename))
```

Comment:
both are binread now?

Reply:
Ahem, while this of course works this was not the goal. 
That's why we have review - sorry about that.
Fixed.